packages/core/src/mpi/normalize-patient.ts

```diff
@@ -73,18 +77,12 @@ export function normalizeEmail(email: string): string {
 
 /**
  * Normalizes a phone number by removing all non-numeric characters and, if applicable, removing the country code.
+ * @deprecated use `normalizePhoneNumber` from `@metriport/shared` instead.
  * @param phoneNumber - The phone number to be normalized.
  * @returns The normalized phone number as a string.
  */
-function normalizePhoneNumber(phoneNumber: string): string {
-  const normalizedNumber = phoneNumber.replace(/\D/g, "");
-
-  if (normalizedNumber.startsWith("1") && normalizedNumber.length === 11) {
-    // applies to US and Canada numbers only
-    return normalizedNumber.substring(1);
-  }
-
-  return normalizedNumber;
+export function normalizePhoneNumber(phoneNumber: string): string {
+  return normalizePhoneNumberFromShared(phoneNumber);
```

Comment:
why not `autofix` here?

Reply:
The idea was to not autofix cx input data, so if they provide a phone number like `1234567890 ext 123` we return the original number (it should fails on the route), whereas with `autofix` we would remove the extension.

At the end of the day, there should be no phone number in our DB that requires "autofix", since they should all be 10 digits.

Reply:
Yea, I think for "normalization" the goal is to make things comparable. So the idea is that someone enters something, and we normalize it so that it *might* match something we have internally (same thing for demo augmentation). Technically for demo augmentation, we then use the normalized value, but I'd argue this isn't actually a good term solution (but was fast).

If someone enters something, normalization should do it's best to make to make it "metriport valid". If it's too hard, it passes back undefined essentially saying "this value can't be used for comparison". If it passes back a value and it matches our internal value, then we're saying "the thing that got passed it is close enough to our thing it works". With a scoring function, they'd need enough close data to pass, so I think this approach works.

Thoughts?

Reply:
I think we should separate things here:
- normalization: make it "metriport valid" - "make it fit"
   - that's useful for data that we either accept or toss away, there's no failing (most likely, only HIE originated data)
- comparison: is it similar enough that we consider being the same?
    - AFAIK, this always compares some input data (cx or HIE) against what we have on the DB, right?
    - If that's true, then we can use normalization on the input data with the approach "make it metriport valid"
    - if not, then that's where the normalization with a parameter to auto fix it or not could be useful
       - or just don't' use normalization

Reply:
So I was talking about the case of using normalization to compare against the DB, which the new normalization function now does correctly (i.e. it always returns a 10 digit string). This allows us to compare incoming data vs. our schema (10 digit string). If the incoming data is less than 10 digits, it automatically does not match, but this is likely okay for now and not in scope here.

Reply:
By failing I simply meant that the normalization function could take a 2 digit phone number and say "this isn't even a phone number" and instead of passing back the phone number, could drop it. But I agree with you, the next step (comparing) can simply be written in way that ensures a 2 digit phone number never matches.